lib/taskana-core/src/test/java/acceptance/task/QueryTasksByWildcardSearchAccTest.java

```diff
@@ -61,6 +61,32 @@ void should_ReturnAllTasks_For_ProvidedSearchValueAndAdditionalParameters() {
     assertThat(foundTasks).hasSize(1);
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void should_ReturnAllTasks_For_CaseInsensitiveValue() {
```

Comment:
As I remember our pattern is "should_xxx_When_xxx" . I would suggest for example "should_ReturnAllTasksCaseInsensitive_When_PerformingWildcardQuery".

Reply:
Done